lib/src/table/table-body/table-body.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, Input, EventEmitter, Output } from "@angular/core";
+import { TableHeaderListItem, TableRowClickedEvent } from "../table.models";
+
+@Component({
+    selector: "[seb-table-body]",
+    templateUrl: "./table-body.component.html",
+})
+export class TableBodyComponent {
+    @Input() headerList: Array<TableHeaderListItem> = [];
+    @Input() rows: Array<any> = [];
```

Comment:
is it possible to make it generic instead of using `any` here?

Reply:
@kherP We discussed it and we don't think it's possible. If you know a way let me know. I will resolve this for now.